src/lib/shim/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "shadow-shim"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+libc = { version = "0.2", default-features = false }
+shadow-shim-helper-rs = { path = "../shadow-shim-helper-rs" }
+shadow_shmem = { path = "../shmem" }
+shadow_tsc = { path = "../tsc" }
+logger = { path = "../logger" }
+
+[build-dependencies]
+bindgen = { version = "0.65.1" }
+cbindgen = { version = "0.24.5" }
+cc = { version = "1.0", features = ["parallel"] }
+shadow-build-common = { path = "../shadow-build-common" }
+system-deps = "6.1"
+
+[package.metadata.system-deps]
+# Keep consistent with the minimum version number in src/CMakeLists.txt
+glib = { name = "glib-2.0", version = "2.32" }
```

Comment:
I don't see this anywhere in `src/CMakeLists.txt`?

Reply:
Should be `/CMakelists.txt`. And was out of date. Updated version numbers and comments.